### ISSUE TITLE
Quiz questions: Clicking options is hard (radio buttons + text label should be in a <label>)

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/ocw-hugo-themes#1251]
## Expected Behavior
In radio button forms, clicking the text label should select the radio button. *Otherwise, click targets are very small, especially on mobile.*

## Current Behavior
In order to select the quiz option, you must click the radio circle itself.

## Steps to Reproduce
1. View https://ocw.mit.edu/courses/15-071-the-analytics-edge-spring-2017/pages/logistic-regression/the-framingham-heart-study-evaluating-risk-factors-to-save-lives/quick-question-225/
2. Click the textual label next to a radio button. *It doesn't select the option, but it should*.

For the correct behavior, see old site: https://old.ocw.mit.edu/courses/sloan-school-of-management/15-071-the-analytics-edge-spring-2017/logistic-regression/the-framingham-heart-study-evaluating-risk-factors-to-save-lives/quick-question-225/

## Possible Solution
The HTML for each option currently looks roughly like this
```html
<div class="multiple-choice-div">
	<input type="radio">
	<span>LABEL TEXT HERE</span>
  	<span>correctness icon</span>
</div>
```
The outer div should be a `<label>`.
